java/core/src/main/java/co/worklytics/psoxy/gateway/LockService.java

```diff
@@ -0,0 +1,8 @@
+package co.worklytics.psoxy.gateway;
+
+public interface LockService {
+
+    boolean acquire(String lockId);
```

Comment:
```suggestion
    boolean acquire(String lockId, long timeoutMs);
```
I'd pass it as param